Library/Formula/objective-caml.rb

```diff
@@ -18,20 +19,18 @@ class ObjectiveCaml < Formula
   end
 
   def install
-    args = %W[
-      --prefix #{HOMEBREW_PREFIX}
-      --mandir #{man}
-      -with-debug-runtime
-    ]
-    args << "-cc" << "#{ENV.cc} #{ENV.cflags}"
-    args << "-aspp" << "#{ENV.cc} #{ENV.cflags} -c"
-    args << "-no-graph" if build.without? "x11"
-
     ENV.deparallelize # Builds are not parallel-safe, esp. with many cores
+
+    # the ./configure in this package is NOT a GNU autoconf script!
+    args = ["-prefix", prefix, "-with-debug-runtime", "-mandir", man]
```

Comment:
This patch breaks OCaml quite badly, since the baked in prefix from `ocamlc -where` has changed to the Cellar location from `/usr/local/bin`.  I'll submit a pull request to change the `prefix` back to `#{HOMEBREW_PREFIX}` as it was before.

Specifically, the `camlp4` libs are no longer found (since it's a separate formula) and most packages fail to install.